cmd/server/main.go

```diff
@@ -391,6 +396,11 @@ func (p *Proxy) runUDSMasterServer(ctx context.Context, o *ProxyRunOptions, s *s
 	if o.mode == "grpc" {
 		grpcServer := grpc.NewServer()
 		client.RegisterProxyServiceServer(grpcServer, s)
+		if o.deleteUDSFile {
```

Comment:
I remember this PR was originally created to address an improper uds socket cleanup in grpc mode, but given the description of this flag shouldn't we also perform the cleanup in http-connect mode?

Reply:
Good catch. Updated.